multivariate-linear-regression/index.js

```diff
@@ -68,12 +61,12 @@ data.loadData().then(async () => {
     }
   });
 
-  await ui.updateStatus('status', `Running on test data`);
-
+  await ui.updateStatus('Running on test data...');
   const result =
       model.evaluate(testData.data, testData.target, {batchSize: BATCH_SIZE});
-
   const testLoss = result.get().toFixed(4);
-
-  await ui.updateStatus('status', `Test set loss: ${testLoss}`);
+  await ui.updateStatus(
+      `Final train-set loss: ${trainLoss};` +
```

Comment:
Probably we can break it into 3 different lines for improving readability. Also, we should round off `trainLoss` and `valLoss`.